code/0_download_era5.py

```diff
@@ -0,0 +1,95 @@
+# This script downloads ERA5 data and organizes where it is stored. 
+# A general function is defined and then called for years 2002-2021. 
+# The following variables are called for: 
+# - Tmin, Tmax, Tavg 2m temperature
+# - Precipitation
+# - Ultraviolet radiation
+# Anna Boser, Jan 28, 2022
+
+from pyprojroot import here
+import cdsapi
+import os
+
+# data_folder = '/home/tcarleton/climate-aggregate/data'
+data_folder = '../data'
```

Comment:
@anna-boser this is not best practice- we should be setting rood directories one time either in an environment or with a root dir chunk at the top of the script so everything is relative to that.

Reply:
Ok I just changed this -- not sure if I did what you meant though. Additionally @traceybit and @saraorofino and I were talking about it a bit today and we think we might just want to define the root in a separate file that we source at the beginning of each script. I haven't done this because I'm not sure how to do it in python but if you think this is a good approach I can do it for all of the R scripts

Reply:
0.5_resmaple_rasters.R in the main branch also does the same thing I was doing in the download scripts so may need to be changed one way or another